code/modules/mob/living/carbon/human/life.dm

```diff
@@ -180,7 +180,7 @@
 			for(var/mob/O in viewers(src, null))
 				if(O == src)
 					continue
-				O.show_message(text("<span class='danger'>[src] starts having a seizure!</span>"), 1)
+				O.show_message("<span class='danger'>[src] starts having a seizure!</span>", 1)
```

Comment:
If you feel like adjusting `show_message` slightly to handle cases where all the msg inputs are blank, you could change this to a visible message and get rid of another `for(var/mob/O in viewers)`

Or maybe some other solution just in `visible_message`